### DISCUSSION TITLE
Scanning Patterns

### DISCUSSION THREAD

**Discussion by lageist** [UNC-Project-Open-AAC/OS-DPI#114] [Help]
@gbishop @sofia-bg @jbizovi @frrancode @aetkie Exploring scanning patterns with 4 groups on 36 location main page and row/column on 9 location pages. OSDPI file [here](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/23dc1bf94583aded08f920fb5673655832c63616/Designs%20In%20Progress/LG_Explore/9_loc_4_pg_core_1022122.osdpi). Sketch [here](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/387e3046fba2c0ee322d75333249b0e4e0baa04e/Designs%20In%20Progress/LG_Explore/9_loc_4_pg_sketch.png)

Exploring scanning patterns with top (group 1) and right (group 2) navigation and content grid(s) (group 3). Some declarative programming problems and/or bugs in the current file [here](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/23dc1bf94583aded08f920fb5673655832c63616/Designs%20In%20Progress/LG_Explore/top_right_nav_102122.osdpi). Sketch [here](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/387e3046fba2c0ee322d75333249b0e4e0baa04e/Designs%20In%20Progress/LG_Explore/top_right_nav_sketch.png).

**Comment by aetkie** [comment]
@lageist I was able to open this take a look around after you re-uploaded and I did a reset. I like the design! Great job. Wondering what your intentions are for the Patterns? I am still trying to learn how to make those work and any insight into your thought process in the design would be helpful.

**Reply by lageist** [reply]
Somehow missed this earlier, sorry @aetkie. Used the above solely for exploration purposes, looking at how to program patterns with groups (beyond row/column). Used the 4 location to simulate one of the partner-assisted scanning formats we used in Project Core, and the other design to explore grouping 3 main portions of the UI (e.g., navigation to words, navigation to quick phrases, display of words or phrases).
